templates/example_cell_clustering.ipynb

```diff
@@ -11,13 +11,15 @@
    "cell_type": "markdown",
```

Comment:
<p>Colors here should correspond to metacluster colors in the heatmap. Not viridis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Yeah for some reason I thought that you used a <code>viridis</code>  color scheme, must have confused it with a different visualization. I can easily change this back to the original color scheme.</p>